src/Symfony/Component/Config/Tests/Resource/ReflectionClassResourceTest.php

```diff
@@ -117,49 +117,49 @@ public function testHashedSignature(bool $changeExpected, int $changedLine, ?str
-        yield [true, 5, '#[Foo(new MissingClass)] public $pub = [];'];
-        yield [true, 6, '/** prot docblock */'];
-        yield [true, 7, 'private $prot;'];
-        yield [false, 8, '/** priv docblock */'];
-        yield [false, 9, 'private $priv = 123;'];
-        yield [true, 10, '/** pub docblock */'];
-        yield [true, 11, 'public function pub(...$arg) {}'];
-        yield [true, 11, 'public function pub($arg = null): Foo {}'];
-        yield [false, 11, "public function pub(\$arg = null) {\nreturn 123;\n}"];
-        yield [true, 12, '/** prot docblock */'];
-        yield [true, 13, 'protected function prot($a = [123]) {}'];
-        yield [true, 13, '#[Foo] protected function prot($a = []) {}'];
-        yield [true, 13, 'protected function prot(#[Foo] $a = []) {}'];
-        yield [true, 13, '#[Foo(new MissingClass)] protected function prot($a = []) {}'];
-        yield [true, 13, 'protected function prot(#[Foo(new MissingClass)] $a = []) {}'];
-        yield [false, 14, '/** priv docblock */'];
-        yield [false, 15, ''];
+        yield [false, 0, "// line change\n\n", '1'];
+        yield [true, 0, '/** class docblock */', '2'];
+        yield [true, 0, '#[Foo]', '3'];
+        yield [true, 0, '#[Foo(new MissingClass)]', '4'];
+        yield [true, 1, 'abstract class %s', '5'];
+        yield [true, 1, 'final class %s', '6'];
+        yield [true, 1, 'class %s extends Exception', '7'];
+        yield [true, 1, 'class %s implements '.DummyInterface::class, '8'];
+        yield [true, 3, 'const FOO = 456;', '9'];
+        yield [true, 3, 'const BAR = 123;', '10'];
+        yield [true, 4, '/** pub docblock */', '11'];
+        yield [true, 5, 'protected $pub = [];', '12'];
+        yield [true, 5, 'public $pub = [123];', '13'];
+        yield [true, 5, '#[Foo(new MissingClass)] public $pub = [];', '14'];
+        yield [true, 6, '/** prot docblock */', '15'];
+        yield [true, 7, 'private $prot;', '16'];
+        yield [false, 8, '/** priv docblock */', '17'];
+        yield [false, 9, 'private $priv = 123;', '18'];
+        yield [true, 10, '/** pub docblock */', '19'];
+        yield [true, 11, 'public function pub(...$arg) {}', '20'];
+        yield [true, 11, 'public function pub($arg = null): Foo {}', '21'];
+        yield [false, 11, "public function pub(\$arg = null) {\nreturn 123;\n}", '22'];
+        yield [true, 12, '/** prot docblock */', '23'];
+        yield [true, 13, 'protected function prot($a = [123]) {}', '24'];
+        yield [true, 13, '#[Foo] protected function prot($a = []) {}', '25'];
+        yield [true, 13, 'protected function prot(#[Foo] $a = []) {}', '26'];
+        yield [true, 13, '#[Foo(new MissingClass)] protected function prot($a = []) {}', '27'];
+        yield [true, 13, 'protected function prot(#[Foo(new MissingClass)] $a = []) {}', '28'];
+        yield [false, 14, '/** priv docblock */', '29'];
+        yield [false, 15, null, '30'];
```

Comment:
worth using `++$i` to make it easier to maintain?

Reply:
Or a hash of [`TestCase::nameWithDataSet`](https://github.com/sebastianbergmann/phpunit/blob/0441c6da39bd9dc477206ae0591b678c7e9da0aa/src/Framework/TestCase.php#L396C27-L396C42)

Reply:
Done while merging